### ISSUE TITLE
Pop-up for accented vowels and special chars gets out of screen

### ISSUE THREAD

**Issue by RigacciOrg** [VladThodo/behe-keyboard#24]
If I long-tap on a letter which is on the left side of the screen (e.g. the "a" key), the pop-up for inserting accented letter, etc. brings up. Unfortunately it falls out of the screen on the left side, so I cannot select some of the letters avaliable.
Using BeHe Keyboard 1.0.5 from F-Droid, into an Android 6.0 chinese phone.

**Comment by VladThodo** [comment]
Could you tell me many accents are displayed for the "a" key?

**Comment by RigacciOrg** [comment]
I see only 4 variant: á (a-caute), ǎ (a-caron), ä and â. May be they are them all? I was expecting the à (a-grave)...

**Comment by VladThodo** [comment]
Yes, currently that's all. I will add some more in the future update. Any other accents that you need and the keyboard doesn't have?

**Comment by RigacciOrg** [comment]
I can list what is present on a standard Italian keyboard :-):  òàùèìùéç°§£. May be all the vowels, with both acute and grave accents are the most important.

**Comment by hThoreau** [comment]
Please add 'ã' and 'õ' as well. These are necessary to make use of the Portuguese language.

**Comment by RigacciOrg** [comment]
So it seems that this is not a bug: popup is not off-screen, but just it needs more accented vowels, etc.
So may be we can merge this bug into #21 ?
